bindings/pyroot/pythonizations/python/ROOT/_numbadeclare.py

```diff
@@ -276,6 +287,21 @@ def inner(func, input_types=input_types, return_type=return_type, name=name):
         else:
             pywrapper_return = "return r"
 
+        # bind the arguments that are passed as void* pointers to the correct type
+        pywrapper_bind_lines = "\n    ".join(
+            [
+            "import cppyy" if any(
+                not is_container_type(t) and map_cpp_type(t, as_ctypes=True) == ctypes.c_void_p
+                for t in input_types
+            ) else "",
```

Comment:
```suggestion
            "import cppyy",
```
Python caches imported modules, so if `cppyy` is already imported then `"import cppyy"` has no relevant overhead. And when you use ROOT, it is already imported.

I would consider just always doing `"import cppyy"` to simplify the logic

Reply:
I'll do that thanks